boards/doc.txt

```diff
@@ -17,6 +17,55 @@
+| esp32-wroom-32    | Prototyping   | +++           | ✗                     | ✔ (WiFi, BLE, custom)             | ✗                 | Custom            | - (1 button)                                      | +++                   | +++           | Requires proprietary software |
+| microbit-v2       | Education     | ++            | ✔                     | ✔ (802.15.4, BLE, custom)         | ✗                 | Custom            | +++ (6 buttons, LED matrix, mic, speaker, IMO)    | --                    | ++            | Good education board          |
+| nrf52840dk        | Development   | ++            | ✔                     | ✔ (802.15.4, BLE, custom)         | +++ (Uno, Mega)   | Custom            | + (4 buttons, 4 LEDs)                             | +                     | +             | Excellent boarder router      |
+| nrf52840dongle    | Prototyping   | ++            | ✗                     | ✔ (802.15.4, BLE, custom)         | ✗                 | Custom            | o (1 buttons, 4 LEDs)                             | ++                    | ++            | Good wireless dev board       |
+| nucleo-32         | Development   | -/o/+/++      | ✔                     | ✗                                 | +   (Nano)        | Custom            | o (1 buttons, 1 LED)                              | +                     | ++            | Good bang for the buck        |
+| nucleo-64         | Development   | o/+/++        | ✔                     | ✗                                 | +   (Uno)         | Custom            | o (1 buttons, 1 LED)                              | +                     | ++            | Good bang for the buck        |
+| nucleo-144        | Development   | +/++/+++      | ✔                     | (✔) (some have Ethernet)          | ++  (Uno, Mega)   | Custom            | + (1 buttons, 3 LEDs)                             | +                     | ++            | Good bang for the buck        |
+| samr21-xpro       | Development   | +             | ✔                     | ✔ (802.15.4)                      | ✗                 | Custom            | o (1 buttons, 1 LED)                              | +                     | --            | Too expensive                 |
+| samr34-xpro       | Development   | ++            | ✔                     | ✔ (LoRa)                          | ✗                 | Custom            | o (1 buttons, 2 LEDs)                             | +                     | ---           | Got a spare kidney to sell?   |
+| weact-f4x1cx      | Prototyping   | ++            | ✗                     | ✗                                 | ✗                 | Custom            | + (1 buttons, 1 LED, SPI flash)                   | +++                   | +++           | Excellent bang for the buck   |
+
+@note   Only boards with mature RIOT support are included above
```

Comment:
Maybe add something like "This list was last updated May 2023"?

Reply:
Done